Rubberduck.Core/UI/Refactorings/ExtractInterface/ExtractInterfacePresenter.cs

```diff
@@ -13,13 +12,7 @@ internal class ExtractInterfacePresenter : RefactoringPresenterBase<ExtractInter
 
         public override ExtractInterfaceModel Show()
         {
-            if (Model.TargetDeclaration == null)
-            {
-                return null;
-            }
-
-            var model = base.Show();
-            return DialogResult != RefactoringDialogResult.Execute ? null : model;
+            return Model.TargetDeclaration == null ? null : base.Show();
```

Comment:
This fixes a bug where cancelling the dialog would extract the interface anyway, right?

Reply:
Was there such a bug?

The way it is now, canceling the dialog will cause a `RefactoringAbortedException` to be thrown.

The null check here really is just a safeguard. The refactoring should never construct a model whose target is `null`.

Reply:
Not sure if there ever was an issue for it - noticed when I x'd out of the dialog after taking a screenshot of the icon for a new (unrelated) issue, and the refactoring went on to extract an interface with no members.